kg_microbe/transform.py

```diff
@@ -49,11 +37,11 @@
     MADIN_ETAL: MadinEtAlTransform,
     RHEAMAPPINGS: RheaMappingsTransform,
     BACTOTRAITS: BactoTraitsTransform,
-    UNIPROT_HUMAN: UniprotHumanTransform,
-    CTD: CTDTransform,
-    DISBIOME: DisbiomeTransform,
-    WALLEN_ETAL: WallenEtAlTransform,
-    UNIPROT_FUNCTIONAL_MICROBES: UniprotFunctionalMicrobesTransform,
+    #UNIPROT_HUMAN: UniprotHumanTransform,
+    #CTD: CTDTransform,
+    #DISBIOME: DisbiomeTransform,
+    #WALLEN_ETAL: WallenEtAlTransform,
+    #UNIPROT_FUNCTIONAL_MICROBES: UniprotFunctionalMicrobesTransform,
 }
```

Comment:
Do we want to remove all of these data sources from the "data source transform index" (`transform.py`)?